src/Views/Form/Templates/Sequoia/assets/js/form.js

```diff
@@ -379,6 +399,11 @@
 	 * @param {number} formID Form ID
 	 */
 	function refreshPaymentInformationSection( ev, response, formID ) {
+		// If mailchimp is opted in by default, add border on load
```

Comment:
What about constant contact? Is that not an option?

Reply:
Added support for constant contact: https://github.com/impress-org/givewp/commit/bddd8865e2ee87773bb1bb9884a0a47fa5be120c